### PR TITLE
Improve X timeout message

### PR DESCRIPTION
**Pull request by VladimirSlavik**
Suggested by @jstodola.

### CODE REVIEW

File: pyanaconda/display.py [hunk 1]
```diff
@@ -50,6 +50,17 @@
 stdout_log = get_stdout_logger()
 
 
+X_FAILURE_MESSAGE = \
+    "X startup failed, falling back to text mode. There are multiple ways to help this issue:\n" \
+    "* Do not load the stage2 image over a slow network link.\n" \
+    "* Wait for the X server startup longer with the inst.xtimeout=<SECONDS> boot option. " \
+    "The default is 60 seconds.\n" \
```
Comment (poncovka) [inline pyanaconda/display.py:57]: Could add two spaces before the line, so it has the same indentation as the one above?

File: pyanaconda/display.py [hunk 2]
```diff
@@ -50,6 +50,17 @@
...
+    "* Load the stage2 image into memory with the rd.live.ram boot option to decrease access " \
+    "time.\n" \
```
Comment (poncovka) [inline pyanaconda/display.py:59]: Same here.

**Review by poncovka — APPROVED** [review]
Looks good to me, but you will need a bug for this branch.

File: pyanaconda/display.py (hunk 1 above)
Comment (VladimirSlavik) [reply pyanaconda/display.py:57]: I'm afraid you expect too much :) It's literally just bullet points that get wrapped by default, no padding, nothing. But... challenge accepted!

Comment (poncovka) [reply pyanaconda/display.py:57]: Oh, I thought that each line ends with `\n`. Never mind.

**Comment by VladimirSlavik** [comment]
Update: Changed to wrap automatically.

Changing to draft + blocked until it's clear if this is the direction @jstodola wanted.

File: pyanaconda/display.py [hunk 3]
```diff
@@ -50,6 +50,17 @@
 stdout_log = get_stdout_logger()
 
 
+X_FAILURE_MESSAGE = \
+    "X startup failed, falling back to text mode. There are multiple ways to help this issue:\n" \
```
Comment (M4rtinK) [inline pyanaconda/display.py]: "There are multiple ways to help this issue" sounds to me like the bullet points above help _cause_ the issue.

Instead I recommending changing the formulation to something like this:

"There are multiple ways that might help avoiding this issue"

File: pyanaconda/display.py [hunk 4]
```diff
@@ -327,7 +327,13 @@ def setup_display(anaconda, options):
             do_startup_x11_actions()
         except (OSError, RuntimeError) as e:
             log.warning("X startup failed: %s", e)
-            stdout_log.warning(X_FAILURE_MESSAGE)
+            stdout_log.warning("X startup failed, falling back to text mode. There are multiple "
+                               "ways to help this issue:")
```
Comment (M4rtinK) [inline pyanaconda/display.py:331]: Same suggestion - change to "help resolve this issue".

**Review by M4rtinK — APPROVED** [review]
Looks good to me, some minor wording changes suggested. :)

**Comment by VladimirSlavik** [comment]
Possibly superseded by #3304 which integrates this upstream and could be backported.

**Comment by VladimirSlavik** [comment]
Instead of this, let's backport upstream #3304.

File: pyanaconda/display.py (hunk 4 above)
Comment (VladimirSlavik) [reply pyanaconda/display.py:331]: Thank you, used in subsequent PR.

**Pull request closed without merging**